Gemfile_dogstats_v4.lock

```diff
@@ -0,0 +1,83 @@
+PATH
```

Comment:
This file is identical to `Gemfile.lock` except for the version of `dogstatsd-ruby`